lcls-twincat-motion/Library/POUs/Motion/States/FB_PositionStateMove.TcPOU

```diff
@@ -114,15 +122,24 @@ fbMotionRequest(
     bBusy => bBusy,
     bDone => bDone);
 
+rtReset(CLK:=bReset);
+IF rtReset.Q THEN
+    nLatchAllowErrorID := 0;
+END_IF
+
 // Inject custom error if we can't move because of bMoveOk or bValid
-IF bExecute AND NOT bAllowMove THEN
+IF nLatchAllowErrorID <> 0 OR (bExecute AND NOT bAllowMove) THEN
     bError := TRUE;
-    IF stPositionState.bValid THEN
-        nErrorId := 16#7901;
+    IF nLatchAllowErrorID <> 0 THEN
+        nErrorID := nLatchAllowErrorID;
+    ELSIF stPositionState.bValid THEN
+        nErrorID := 16#7901;
```

Comment:
A comment on these magic numbers would be appreciated (or eventually moved to an enum as we *may* have previously chatted about, I think)

Reply:
I thought I'd done the background work for this and forgot to apply it here, but the reality is that I only implemented enums so far for the FF types. The custom error codes are reflected only the error message writer FB, this needs some better organization as you've said before.
https://github.com/pcdshub/lcls-twincat-motion/blob/master/lcls-twincat-motion/Library/DUTs/E_MotionFFType.TcDUT
https://github.com/pcdshub/lcls-twincat-motion/blob/master/lcls-twincat-motion/Library/POUs/Motion/Utils/F_MotionErrorCodeLookup.TcPOU

Reply:
My intuition is that this calls for a new enum specifically for custom error types- not to rehash all the built-in errors.

Reply:
I think the "custom error code enum" makes a lot of sense. This certainly could be in a follow-up PR (if at all)

Reply:
Handled in https://github.com/pcdshub/lcls-twincat-motion/pull/205/commits/1170c26791cf8bf6f5f0c0f8a2e1011ab3e221c7